.github/workflows/lint-vet-gofmt-staticcheck-analysis.yml

```diff
@@ -18,6 +18,7 @@ jobs:
           go mod tidy
           go get golang.org/x/lint/golint
           echo "$HOME/go/bin" >> $GITHUB_PATH
+          go install honnef.co/go/tools/cmd/staticcheck@latest
 
       - name: Lint Analysis
```

Comment:
What do you think excluding Golint work since it's deprecated?

Reply:
As I wrote earlier:

_I propose a lint to not be excluded as it is also useful and used in the Go report system and we have the lowest indicators on it. [golint->76%](https://goreportcard.com/report/github.com/lf-edge/edge-home-orchestration-go)._

But I think that when we fix them then we will exclude lint